BookingsAPI/Bookings.DAL/Commands/UpdateSuitabilityAnswersCommand.cs

```diff
@@ -49,7 +49,17 @@ public async Task Handle(UpdateSuitabilityAnswersCommand command)
             {
                 throw new ParticipantNotFoundException(command.ParticipantId);
             }
-            participant.AddSuitabilityAnswers(command.SuitabilityAnswers);
+
+            if (participant.Questionnaire == null)
```

Comment:
I believe we should always have a questionnaire, otherwise we can throw a 404 maybe

Reply:
I think we don't need always questionnaire, as judges have not any

Reply:
But in that case we shouldn't be able to update it either. In this case you can . create a new questionnaire for judges as well

Reply:
We would not update judge, but why we need to create record Questionnaire if we don't need to, just take space in database?